### DISCUSSION TITLE
Add --output switch to write generated credentials to destination other than stdout

### DISCUSSION THREAD

**Discussion by fenech** [ory/oathkeeper#869] [Ideas]
I ran into an issue earlier, when trying to use the scratch-based image in an initContainer. The command `oathkeeper credentials generate` sends its output to `os.Stdout`, which means that it depends on a parent shell to write the output to a file.

https://github.com/ory/oathkeeper/blob/master/cmd/credentials_generate.go#L46

My current workaround is to use the Alpine-based image instead, so that I can redirect the output of the command. I think it would be convenient to add an option like `--output`, so that a destination path could be passed, and the JWKs written to the path.

**Comment by vinckr** [comment]
Thanks for the idea @fenech !

Maybe this would be nice to include in [Oathkeeper Next-Gen](https://github.com/ory/oathkeeper/issues/441)?
